src/server/routes/github.js

```diff
@@ -1,11 +1,14 @@
 import { Router } from 'express';
 import { json } from 'body-parser';
 
-import { createWebhook } from '../handlers/github';
+import { createWebhook, listRepositories } from '../handlers/github';
 
 const router = Router();
 
 router.use('/github/webhook', json());
 router.post('/github/webhook', createWebhook);
 
+router.use('/github/repos', json());
+router.get('/github/repos', listRepositories);
```

Comment:
Please document new routes in `docs/routes.md` (I know not all the existing ones are documented, but we should at least keep up with new ones).